docs/start/getting_started.rst

```diff
@@ -197,6 +198,93 @@ In production, content is served through CherryPy. Static assets must be pre-bui
+ 
+    * Set ``KOLIBRI_PEX_URL`` to string ``default`` to run latest pex from Kolibri download page
+    * Set ``KOLIBRI_PEX_URL`` to something like ``http://host.org/nameof.pex``
+    * Set ``KOLIBRIBUILD_PEX_PATH`` to something like ``/kolibribuild/nameof.pex``
+    * ``KOLIBRI_RUN_MODE``: set in Dockerfile
+    * ``KOLIBRI_PROVISIONDEVICE_FACILITY``: if this environment variable is set
+      the entrypoint script will run the provision device an setup a facility
+      with this name. The ``KOLIBRI_LANG`` environment variable and the following
+      other environment variables will be used in the process:
+        
+        * ``KOLIBRI_PROVISIONDEVICE_PRESET``: defaults to ``formal``, with the other options being ``nonformal`` and ``informal``
+        * ``KOLIBRI_PROVISIONDEVICE_SUPERUSERNAME``: default ``devowner``
+        * ``KOLIBRI_PROVISIONDEVICE_SUPERUSERPASSWORD``: default ``admin123``
+
+    * ``KOLIBRI_HOME``: default ``/kolibrihome``
+    * ``KOLIBRI_HTTP_PORT``: default ``8080``
+    * ``KOLIBRI_LANG``: default ``en``
+    * ``CHANNELS_TO_IMPORT``: comma-separated list of channel IDs (not set by default)
+
+
+Building a pex file
+^^^^^^^^^^^^^^^^^^^
+When simply testing things out or reviewing a pull request, the easiest way to
+obtain a pex file is to get the link from the buildkite assets link that is present
+for every git branch and every pull request. This is the approach we recommend in
+combination with the ``demoserver`` approach for running described in the next section. 
+
+However, if you want to build and run a pex from the Kolibri code in your current
+local source files without relying on the github and the buildkite integration,
+you can run the following commands to build a pex file in ``kolibribuild/``:
+
+.. code-block:: bash
+
+  make dockerbuildbase      # only needed first time
+  make dockerbuild
+
+Run the the resulting pex using the ``demoserver`` approach described below.
+
+
+Starting a demoserver
+^^^^^^^^^^^^^^^^^^^^^
+You can start a Kolibri instance running any pex file by setting the appropriate
+environment variables in your local copy of `env.list` then running the commands:
+
+.. code-block:: bash
+
+  make dockerbuildbase      # only needed first time
+  make dockerdemoserver
+
+The choice of pex file can be controlled by setting environment variables in the
+file ``env.list``:
+
+ * Set ``KOLIBRI_PEX_URL`` to string ``default`` to run the latest pex from Kolibri download page
+ * Set ``KOLIBRI_PEX_URL`` to something like ``http://host.org/nameof.pex``
+ * Set ``KOLIBRIBUILD_PEX_PATH`` to something like ``/kolibribuild/nameof.pex``
+
+
+ 
+ Starting a devserver
+ ^^^^^^^^^^^^^^^^^^^^
+ Use these commands to start the Kolibri devserver running inside a container:
+
+ .. code-block:: bash
+
+   make dockerbuildbase      # takes a long time (go get a coffee)
+   make dockerdevserver      # takes a few mins to run pip install -e + webpcak build
```

Comment:
You used `dockerdevserver` here, but it's `dockerdeveserver` in the Makefile! I prefer the former.